src/Framework/PaymentGateways/SubscriptionModule.php

```diff
@@ -4,7 +4,9 @@
 
 use Give\Framework\PaymentGateways\Contracts\Subscription\SubscriptionAmountEditable;
 use Give\Framework\PaymentGateways\Contracts\Subscription\SubscriptionDashboardLinkable;
+use Give\Framework\PaymentGateways\Contracts\Subscription\SubscriptionPausable;
 use Give\Framework\PaymentGateways\Contracts\Subscription\SubscriptionPaymentMethodEditable;
+use Give\Framework\PaymentGateways\Contracts\Subscription\SubscriptionResumable;
```

Comment:
Do we need this still?

Reply:
No. That's a leftover.

Reply:
Just checking, everything seems to be in order with the UI and this work. This should be good to go! Would you prefer I make an epic for the UI & this or would you rather work off dev?